### DISCUSSION TITLE
Using the Thermostat module to heat water for beer brewing?

### DISCUSSION THREAD

**Discussion by boc-tothefuture** [arendst/Tasmota#11684] [Q&A] [answered]
I am considering using the thermostat module for Tasmota to assist with beer brewing.

My plan would be to take an immersion heater [like this](https://www.amazon.com/Immersion-Electric-Stainless-Protective-Submersible/dp/B08GKGK4HR) and connect it to a Tasmota, coupled with a one-wire temp sensor placed in the water.

I would need to get the water up to around 160 to 170 degrees.  I wouldn't particularly care how long it takes to get there.  My plan would be to have everything setup the night before, set a timer from my home automation system to tell it when to start (maybe 3 hours or so before I wake up) and then have it kick on to hit up the water.

I think I would need to set the thermostat to PI mode only.  

It seems workable, but wondering if there is anything I am missing about how the tasmota system works that might be an issue?

I have a couple of Tasmota devices around my house (light bulbs, and switches) so I am familiar with the platform in general, but not the thermostat module.

**Comment by ascillato2** [comment, marked as answer]
Please, address this to the [Tasmota Support Chat](https://discord.gg/Ks2Kzd4). The chat is a better and more dynamic channel for helping you.

Remember to read the [Contributing Guideline and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md). Thanks.

---------------------------

### Support Information 
See [Docs](https://tasmota.github.io/docs/) for more information.
See [Chat](https://discord.gg/Ks2Kzd4) for more user experience.
See [Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
